eventhubs-capture-synapse-delta/create-solution.sh

```diff
@@ -0,0 +1,205 @@
+shift $((OPTIND-1))
+
+if [[ -z "$PREFIX" ]]; then
+    echo "Enter a name for this deployment."
+    usage
+fi
+
+if [[ -z "$SQLPASSWORD" ]]; then
+    echo "Enter a SQL password for the Sparkpool"
+    usage
+fi
+
+# 10000 messages/sec
+if [ "$TESTTYPE" == "10" ]; then
+    export EVENTHUB_PARTITIONS=12
+    export EVENTHUB_CAPACITY=12
+    export SIMULATOR_INSTANCES=5
+    export SPARK_NODE_COUNT=5
+    export SPARK_NODE_SIZE=Large
+fi
+
+# 5000 messages/sec
+if [ "$TESTTYPE" == "5" ]; then
+    export EVENTHUB_PARTITIONS=8
+    export EVENTHUB_CAPACITY=6
+    export SIMULATOR_INSTANCES=3
+    export SPARK_NODE_COUNT=3
+    export SPARK_NODE_SIZE=Medium
+fi
+
+# 1000 messages/sec
+if [ "$TESTTYPE" == "1" ]; then
+    export EVENTHUB_PARTITIONS=2
+    export EVENTHUB_CAPACITY=2
+    export SIMULATOR_INSTANCES=1
+    export SPARK_NODE_COUNT=3
+    export SPARK_NODE_SIZE=Small
+fi
+
+# last checks and variables setup
+if [ -z ${SIMULATOR_INSTANCES+x} ]; then
+    usage
+fi
+
+export RESOURCE_GROUP=$PREFIX
+
+# remove log.txt if exists
+rm -f log.txt
+
+echo "Checking pre-requisites..."
+
+source ../assert/has-local-az.sh
+source ../assert/has-local-jq.sh
+source ../assert/has-local-zip.sh
+source ../assert/has-local-dotnet.sh
+
+declare STORAGE_EVENT_QUEUE="blob-events"
+
+echo
+echo "Streaming at Scale with Azure Synapse and Delta"
+echo "=================================================="
+echo
+
+echo "Steps to be executed: $STEPS"
+echo
+
+echo "Configuration: "
+echo ". Resource Group  => $RESOURCE_GROUP"
+echo ". Region          => $LOCATION"
+echo ". EventHubs       => TU: $EVENTHUB_CAPACITY, Partitions: $EVENTHUB_PARTITIONS"
+echo ". Simulators      => $SIMULATOR_INSTANCES"
+echo
+
+echo "Deployment started..."
+echo
+
+echo "***** [C] Setting up COMMON resources"
+
+    export AZURE_STORAGE_ACCOUNT=$PREFIX"storage"
+    export AZURE_STORAGE_ACCOUNT_GEN2=$PREFIX"storhfs"
+
+    RUN=`echo $STEPS | grep C -o || true`
+    if [ ! -z "$RUN" ]; then
+        source ../components/azure-common/create-resource-group.sh
+        source ../components/azure-storage/create-storage-hfs.sh
+        source ../components/azure-storage/setup-storage-event-grid.sh
+    fi
+echo
+
+echo "***** [I] Setting up INGESTION"
+
+    export EVENTHUB_NAMESPACE=$PREFIX"eventhubs"
+    export EVENTHUB_NAME="streamingatscale-$EVENTHUB_PARTITIONS"
+    export EVENTHUB_CAPTURE=True
+
+    RUN=`echo $STEPS | grep I -o || true`
+    if [ ! -z "$RUN" ]; then
+        source ../components/azure-event-hubs/create-event-hub.sh 
+    fi
+echo
+
+echo "***** [P] Setting up PROCESSING"
+
+    export ADB_WORKSPACE=$PREFIX"synapse"
+    export ADB_TOKEN_KEYVAULT=$PREFIX"kv" #NB AKV names are limited to 24 characters
+    export BLOB_FILE_FORMAT="avro"
+
+    RUN=`echo $STEPS | grep P -o || true`
+    if [ ! -z "$RUN" ]; then
+        echo "Setting up processing. Currently there is no processing layer."
+        source ../components/azure-synapse/create-synapse.sh $SQLPASSWORD
```

Comment:
Getting worrying warning messages:

```
***** [P] Setting up PROCESSING
Setting up processing. Currently there is no processing layer.
Creating Azure Synapse Workspace agsynapse-synwkspc
Parameter --repository-type is missing, the following arguments are ignored: --host-name ,--account-name ,--collaboration-branch ,--repository-name ,--root-folder--project-name ,--tenant-id. Repository configuration will not work.
```

Also, JSON output should be silenced with `-o none`:
```
{
  "adlaResourceId": null,
  "azureAdOnlyAuthentication": null,
  "connectivityEndpoints": {
```